easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb

```diff
@@ -0,0 +1,55 @@
+# Built with EasyBuild version 3.1.1 on 2017-04-21_13-11-20
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2017.02'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
+buildopts = "all"
+
+parallel = 1
+
+files_to_copy = [(['src/IMB-*'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/IMB-EXT', 'bin/IMB-IO', 'bin/IMB-MPI1', 'bin/IMB-NBC', 'bin/IMB-RMA'],
+    'dirs': [],
+}
+
+moduleclass = 'perf'
+
+# Build statistics
```

Comment:
please remove this block (we should make `--new-pr` take care of this actually, see https://github.com/hpcugent/easybuild-framework/issues/1674